Makefile

```diff
@@ -176,6 +176,17 @@ build-test:
 	@test/build/test-configure.sh
 
 
+.PHONY: mango-on-travis
```

Comment:
If you take the latter approach, you'll want to kill off the `./dev/run` process. If you collapsed all of the steps to a single line, you could add a `kill %1` at the end; otherwise you'll have to go fishing for the pid somehow.